03-use-cases/01-finance/retail-banking/main.ipynb

```diff
@@ -0,0 +1,1931 @@
+{
```

Comment:
<p>Please add markdown cells explaining each step. Give some details about what <code># Set explicity data types, others are inferred</code> means here.</p><p> </p><p>For the notebook gallery, if a comment is needed to make the code understandable, it's generally better to add a markdown cell instead. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/atoti/atoti/pull/824/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
<p>Hmmm... These steps are repeated from cells 3-8, so not sure if it would look great essentially repeating the same thing over and over again. This being said, I did modify the note in the markdown cell to say the following, which provides the rationale for the next few cells.</p><p> </p><pre class="ql-syntax" spellcheck="false">💡&nbsp;Note:&nbsp;We should be mindful of the (numerical) columns that we intend to use as hierarchies instead of measures (e.g. a User ID does not need to be aggregated!). For this reason, we can explicitly set specific Atoti Table columns and associated data types. See&nbsp;https://docs.atoti.io/latest/api/atoti.Session.read_csv.html#atoti.Session.read_csv&nbsp;for more info.
</pre>

Reply:
This is done!

Reply:
This is done!